lib/utils/getCommandSuggestion.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const _ = require('lodash');
+const BbPromise = require('bluebird');
+const levenshtein = require('fast-levenshtein');
+
+const getCollectCommandWords = (commandObject, commandWordsArray) => {
+  const wordsArray = _.isArray(commandWordsArray)
+    && !_.isEmpty(commandWordsArray) ? commandWordsArray : [];
+  _.forEach(commandObject, (commandChildObject, commandChildName) => {
```

Comment:
You must use `return BbPromise.each(commandObject, ...)` here instead of `_.forEach()` because you return promises.
Never return promises from within a synchronous loop.